matlab/reco.m

```diff
@@ -20,7 +20,7 @@
 S = h5read(filenameSM, '/measurement/data');
 
 % reinterpret as complex numbers
-S = squeeze(S(1,:,:,:,:) + 1i*S(2,:,:,:,:));
+S = complex(S.r,S.i);
```

Comment:
Yes it is that simple. Reco worked as far as I can tell. (Left and right side looked approx. the same)
Checked with the data download in the morning. Checked with HDF5 that it is using a compound type because i needed the names of the compound fields. MATLAB automatically converts any compound type found in a HDF5 file to a corresponding struct. (unfortunally they do not supply a easy way for the other (write) direction)

For Python it should also be that simple (removing 1 line of code) but i currently do not have a working Python version with all the needed packages to run the script (Installing Anaconda did not solve that). 
Just remove the line:
`S = S[:,:,:,:,:].view(complex128).squeeze()` because S should already be correct